src/patternfly/components/Login/login.scss

```diff
@@ -76,7 +76,6 @@
   --pf-c-login__main-footer-links--PaddingRight: var(--pf-global--spacer--3xl);
   --pf-c-login__main-footer-links--PaddingBottom: var(--pf-global--spacer--xl);
   --pf-c-login__main-footer-links--PaddingLeft: var(--pf-global--spacer--3xl);
-  --pf-c-login__main-footer-links--MarginBottom: calc(var(--pf-c-login__main-footer-links--MarginBottom) * -1);
```

Comment:
Same question as @mcoker had on removing `--pf-c-wizard__toggle-list--MarginBottom` from wizard.scss - did you mean to remove it completely or just move this line below `--pf-c-login__main-footer-links--MarginBottom`?

Reply:
Wait, nevermind - I see this is actually self-referential?

Reply:
Yep. If you look now the margin there just computes to 0 since `--pf-c-login__main-footer-links--MarginBottom` isn't defined anywhere else.

Reply:
As long as this wasn't a mistake where it really should have been assigned to another variable. I'm not sure why it would have been put in there that way. But, I don't know what the original specs were.

Reply:
Yeah I looked at the commit where this was introduced (https://github.com/patternfly/patternfly-next/pull/1281) and looks like a mistake. The layout matches the design in sketch, too.